### ISSUE TITLE
End to End test, which includes authentication

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#1605]
At the moment, all of our tests are blocked by the fact that the user is not logged in. This prevents us from testing comments, which would be nice to cover.

**Comment by jasonLaster** [comment]
Here's a blog post from cypress on what they recommend https://auth0.com/blog/end-to-end-testing-with-cypress-and-auth0/

**Comment by jasonLaster** [comment]
And some auth0 docs

https://auth0.com/docs/architecture-scenarios/implementation/b2b/b2b-launch/b2b-launch-testing
https://community.auth0.com/t/get-token-for-automated-tests/19175

**Comment by jasonLaster** [comment]
lets wait on the new auth approach

**Comment by bhackett1024** [comment]
I'm not sure when the new auth approach is coming, and we're seeing some comments regressions like https://github.com/RecordReplay/devtools/issues/2057 which we ought to have tests for.

With our new playwright integration it seems like it should be easier to write end to end tests that exercise authentication and all the other aspects of the UI which our current test suite has trouble dealing with.  We would need a separate playwright test suite though (different from our playwright-tests repo), but that would be nice to have regardless.

**Comment by jasonLaster** [comment]
We currently have a cypress test that exercises authentication. We should either add it to CI, replace it with a playwright test, or do both.

I'm personally a fan of experimenting with cypress tests in the UX side of replay (comments, collaborators, etc) to understand what cypress is about, but don't feel strongly

**Comment by bhackett1024** [comment]
A cypress test is fine, but it definitely needs to be in CI.
